### ISSUE TITLE
TF Broken with current nightly 2.0

### ISSUE THREAD

**Issue by lc0** [tensorflow/hub#356]
Seems like even simple code is failing from tutorial

```python
IMAGE_SHAPE = (224, 224)

classifier = tf.keras.Sequential([
    hub.KerasLayer(classifier_url, input_shape=IMAGE_SHAPE+(3,))
])
```

Breaks with following error
```
TypeError: Variable is unhashable if Tensor equality is enabled. Instead, use tensor.experimental_ref() as the key.
```

My env had following versions
```python
print(tf.__version__)
print(hub.__version__)
```
```
2.0.0-dev20190822
0.5.0
```

**Comment by lc0** [comment]
Btw, the same is broken with freshly released **2.0.0-rc0** 

Please find a colab for reproducibility
https://colab.research.google.com/drive/1N9heYSp8Cf8TmkZx79LaJWKWeIE01BYI

cc @andresusanopinto

**Comment by vbardiovskyg** [comment]
Hi Sergii, thanks for reporting!

We are aware of the issue, it's caused by interactions with TF on equality operators.

While we release a new 0.6.0 version with a fix, you should be able to workaround using 
```pip install tf-hub-nightly```.

**Comment by lc0** [comment]
@vbardiovskyg  shall we somehow update current docs and tutorials? since currently they are broken if rc is used?

cc @lamberta @dynamicwebpaige

**Comment by vbardiovskyg** [comment]
We've just released 0.6.0, can you please try again?

Which tutorial are you referring to? We've updated the one at https://www.tensorflow.org/hub (not live yet), but maybe there is more?

**Comment by lc0** [comment]
You are right, tutorials by default do not specify rc0 version for tutorials, so only if a person start using rc0 and runs tutorial it would fail.

Just checked and it works fine. Thank you for timely fix!

**Comment by k3it** [comment]
TF 2.0 tutorials now work for me also.  thank you for fixing!

**Comment by kristofgiber** [comment]
I'm confused why are we talking about tfp 0.6, we've been working with 0,7 already without problem until the release of TF 2.0.0-rc where this issue started. Is this now only fixed in 0.6 but not in 0.7?
TypeError: Tensor is unhashable if Tensor equality is enabled. Instead, use tensor.experimental_ref() as the key.

**Comment by kristofgiber** [comment]
After downgrading from tfp 0.7 to 0.6 (with TF2.0.0-rc0) import error:

  File "/usr/local/lib/python3.5/dist-packages/tensorflow_probability/__init__.py", line 78, in <module>
    from tensorflow_probability.python import *  # pylint: disable=wildcard-import
  File "/usr/local/lib/python3.5/dist-packages/tensorflow_probability/python/__init__.py", line 21, in <module>
    from tensorflow_probability.python import bijectors
  File "/usr/local/lib/python3.5/dist-packages/tensorflow_probability/python/bijectors/__init__.py", line 46, in <module>
    from tensorflow_probability.python.bijectors.matveclu import MatvecLU
  File "/usr/local/lib/python3.5/dist-packages/tensorflow_probability/python/bijectors/matveclu.py", line 24, in <module>
    from tensorflow_probability.python.math.linalg import lu_reconstruct
  File "/usr/local/lib/python3.5/dist-packages/tensorflow_probability/python/math/__init__.py", line 22, in <module>
    from tensorflow_probability.python.math.diag_jacobian import diag_jacobian
  File "/usr/local/lib/python3.5/dist-packages/tensorflow_probability/python/math/diag_jacobian.py", line 24, in <module>
    tfe = tf.contrib.eager
AttributeError: module 'tensorflow' has no attribute 'contrib'
